include/tilck/common/boot.h

```diff
@@ -0,0 +1,8 @@
+#define EFI_RUNTIME_CODE MULTIBOOT_MEMORY_RESERVED+1
```

Comment:
Very Good. The only thing is that the names of those two defines is a bit misleading as they appear to be some UEFI constants, while they are totally ours. Therefore, could you please prefix them with "TILCK_BOOT_" so that's clear?